packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksInput.tsx

```diff
@@ -47,7 +66,14 @@ const BlocksInput = React.forwardRef<{ focus: () => void }, BlocksInputProps>(
             </Typography>
             {labelAction && <LabelAction paddingLeft={1}>{labelAction}</LabelAction>}
           </Flex>
-          <BlocksEditor name={name} error={error} ref={forwardedRef} {...editorProps} />
+          <BlocksEditor
+            name={name}
+            error={error}
+            ref={forwardedRef}
+            {...editorProps}
+            ariaLabel={uniqueId}
```

Comment:
Would it make sense just to pass the prop name `aria-labelledBy` instead? 

If not, ariaLabelId is more appropriate imo since this is not an aria-label